test/kpod_build.bats

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "bud-from-scratch" {
+  target=scratch-image
+  kpod build --signature-policy ${BUILDAH_TESTSDIR}/policy.json -t ${target} ${BUILDAH_TESTSDIR}/bud/from-scratch
+  cid=$(buildah from ${target})
+  buildah rm ${cid}
+  buildah rmi $(buildah --debug=false images -q)
+  run buildah --debug=false images -q
+  [ "$output" = "" ]
+}
+
+@test "bud-from-multiple-files-one-from" {
+  target=scratch-image
+  kpod build --signature-policy ${BUILDAH_TESTSDIR}/policy.json -t ${target} -f ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile1.scratch -f ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile2.nofrom
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  cmp $root/Dockerfile1 ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile1.scratch
+  cmp $root/Dockerfile2.nofrom ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile2.nofrom
+  run test -s $root/etc/passwd
+  [ "$status" -ne 0 ]
+  buildah rm ${cid}
+  buildah rmi $(buildah --debug=false images -q)
+  run buildah --debug=false images -q
+  [ "$output" = "" ]
+
+  target=alpine-image
+  kpod build --signature-policy ${BUILDAH_TESTSDIR}/policy.json -t ${target} -f ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile1.alpine -f ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile2.nofrom
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  cmp $root/Dockerfile1 ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile1.alpine
+  cmp $root/Dockerfile2.nofrom ${BUILDAH_TESTSDIR}/bud/from-multiple-files/Dockerfile2.nofrom
+  run test -s $root/etc/passwd
+  [ "$status" -eq 0 ]
+  buildah rm ${cid}
```

Comment:
Can we use kpod rm rather then buildah rm?

Reply:
I think so, will change